### DISCUSSION TITLE
Latest PHP Version

### DISCUSSION THREAD

**Discussion by booblla** [MichaIng/DietPi#6108] [Q&A]
I'm just wondering if the upcoming updates are going to include PHP 8.2.x+ and common modules since PHP 7.4.x has reached its end of life?
I can install it myself after the fact with some [guidance](https://computingforgeeks.com/how-to-install-php-on-debian-linux-2/), but it would be good to have the option to install it during setup.

**Comment by MichaIng** [comment]
See the similar question on our forum: https://dietpi.com/forum/t/php-v7-4-vs-v8-x/15199
- The underlying Debian version is which is responsible for the PHP version. Debian Bullseye ships PHP 7.4.
- Next Debian Bookworm with PHP 8.2 will be released this summer. We already offer images: https://dietpi.com/downloads/images/testing/
- You can also upgrade, but no guarantee that this process goes through without issues and possibly manual config fixes directly or at a later time: https://github.com/MichaIng/DietPi/pull/6103
    - For PHP 8.2, move to `dev` branch before running the upgrade: `G_DEV_BRANCH dev`

**Reply by Joulinar** [reply]
similar question https://github.com/MichaIng/DietPi/issues/5982
